src/foam/u2/view/TableCellFormatter.js

```diff
@@ -144,11 +144,25 @@ foam.CLASS({
     {
       class: 'foam.u2.view.TableCellFormatter',
       name: 'tableCellFormatter',
-      value: function(value) {
-        this.start()
-          .style({'text-align': 'left', 'padding-right': '20px'})
-          .add('$' + (value/100).toFixed(2).replace(/(\d)(?=(\d{3})+\.)/g, '$1,'))
-        .end();
+      value: function(value, obj, axiom) {
+        if ( axiom.unitPropName ) {
+          var unitProp = obj.cls_.getAxiomByName(axiom.unitPropName);
+          var unitId = unitProp.f(obj);
+          obj.__context__.currencyDAO.find(unitId).then((unit) => {
```

Comment:
This should be `unitDAO` since this refinement is for `UnitValue` properties, and currencies should be stored in `unitDAO`.

Reply:
We currently don't have a `unitDAO` yet. @nanoMichal has been working on splitting the DAO's, we can later on change all the `currencyDAO` in nanopay to `unitDAO` where applicable.

Reply:
Ok, then please add a TODO comment so we don't forget to change that afterwards.